### PR TITLE
Dept 890

### PR DESCRIPTION
**Pull request by omahm**

### CODE REVIEW

File: web/modules/custom/dept_migrate/src/LookupHelper.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      return new LookupEntry($this->lookupManager->lookupByDestinationUuid([$uuid]));
+    }
+  }
+
+  /**
+   * Lookup result using source (migration) database.
+   *
+   * @return $this
+   */
+  public function source() {
```
Comment (johangant) [inline web/modules/custom/dept_migrate/src/LookupHelper.php:72]: Handy to have these explicit levers to say which direction the lookup is set to go in :)

File: web/modules/custom/dept_migrate/src/LookupEntry.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+class LookupEntry {
+
+  /**
+   * Lookup data.
+   *
+   * @var false|mixed
+   */
+  protected $item;
+
+  public function __construct(array $lookup_item) {
```
Comment (johangant) [inline web/modules/custom/dept_migrate/src/LookupEntry.php:14]: Safe to assume this is build for a single lookup but has array set to slot in better with other code? Ie: no implementation to handle multiple items at this time, but could be added another day if really needed?

**Review by johangant — APPROVED** [review]
This is great. Makes it a lot clearer about the mechanics of looking up content as well as a cleaner implementation on what we had before

File: web/modules/custom/dept_migrate/src/LookupEntry.php (hunk 2 above)
Comment (omahm) [reply web/modules/custom/dept_migrate/src/LookupEntry.php:14]: I'm just passing the raw results from the lookup service which always returns an array. I had thought about creating a custom collection type but for the time being we don't really need it.

**Pull request merged**